diff --git a/node_modules/@types/nodegit/commit.d.ts b/node_modules/@types/nodegit/commit.d.ts
index 8152eb8..1606c33 100644
--- a/node_modules/@types/nodegit/commit.d.ts
+++ b/node_modules/@types/nodegit/commit.d.ts
@@ -89,7 +89,7 @@ export class Commit {
      *
      *
      */
-    getParents(limit: number, callback?: Function): Promise<Commit[]>;
+    getParents(limit?: number, callback?: Function): Promise<Commit[]>;
     /**
      * Retrieve the commit's parent shas.
      *
diff --git a/node_modules/@types/nodegit/error.d.ts b/node_modules/@types/nodegit/error.d.ts
index c259749..5940953 100644
--- a/node_modules/@types/nodegit/error.d.ts
+++ b/node_modules/@types/nodegit/error.d.ts
@@ -1,5 +1,5 @@
 export namespace Error {
-    const enum ERROR {
+    const ERROR = {
         GITERR_NONE = 0,
         GITERR_NOMEMORY = 1,
         GITERR_OS = 2,
@@ -31,9 +31,9 @@ export namespace Error {
         GITERR_DESCRIBE = 28,
         GITERR_REBASE = 29,
         GITERR_FILESYSTEM = 30
-    }
+    },
 
-    const enum CODE {
+    const CODE = {
         OK = 0,
         ERROR = -1,
         ENOTFOUND = -3,
diff --git a/node_modules/@types/nodegit/rev-walk.d.ts b/node_modules/@types/nodegit/rev-walk.d.ts
index 03781cc..bb6f81b 100644
--- a/node_modules/@types/nodegit/rev-walk.d.ts
+++ b/node_modules/@types/nodegit/rev-walk.d.ts
@@ -3,7 +3,7 @@ import { Oid } from './oid';
 import { Commit } from './commit';
 
 export namespace Revwalk {
-    const enum SORT {
+    const SORT = {
         NONE = 0,
         TOPOLOGICAL = 1,
         TIME = 2,
diff --git a/node_modules/@types/nodegit/tree.d.ts b/node_modules/@types/nodegit/tree.d.ts
index 4546316..da14b5f 100644
--- a/node_modules/@types/nodegit/tree.d.ts
+++ b/node_modules/@types/nodegit/tree.d.ts
@@ -3,7 +3,7 @@ import { TreeEntry } from './tree-entry';
 import { Repository } from './repository';
 import { Object } from './object';
 import { Treebuilder } from './tree-builder';
-import { DiffFile } from './diff-file';
+import { Diff } from './diff';
 import { TreeUpdate } from './tree-update';
 
 export namespace Tree {
@@ -38,11 +38,11 @@ export class Tree {
     /**
      * Diff two trees
      */
-    diff(tree: Tree, callback?: Function): Promise<DiffFile[]>;
+    diff(tree: Tree | undefined, callback?: Function): Promise<Diff>;
     /**
      * Diff two trees with options
      */
-    diffWithOptions(tree: Tree, options?: Object, callback?: Function): Promise<DiffFile[]>;
+    diffWithOptions(tree: Tree | undefined, options?: Object, callback?: Function): Promise<Diff>;
     /**
      * Get an entry at the ith position.
      */
