diff --git a/node_modules/nodegit/dist/nodegit.js b/node_modules/nodegit/dist/nodegit.js
index a292c13..1986dd2 100644
--- a/node_modules/nodegit/dist/nodegit.js
+++ b/node_modules/nodegit/dist/nodegit.js
@@ -16,7 +16,8 @@ try {
     throw ex;
   }
 
-  rawApi = require("../build/Debug/nodegit.node");
+  // a bundler might try to resolve this dynamic require regardless of the result of the if-condition, which will not work if the target does not exist
+  // rawApi = require("../build/Debug/nodegit.node");
 }
 
 var promisify = function promisify(fn) {
