diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index 350d424..cea9a7b 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -549,6 +549,11 @@ module.exports = function(webpackEnv) {
                 'sass-loader'
               ),
             },
+            // for nodegit, also see https://github.com/webpack-contrib/node-loader/issues/12
+            {
+              test: /\.node$/,
+              loader: process.env.NODE_ENV === 'development' ? 'node-loader' : 'native-addon-loader'
+            },
             // "file" loader makes sure those assets get served by WebpackDevServer.
             // When you `import` an asset, you get its (virtual) filename.
             // In production, they would get copied to the `build` folder.
@@ -713,7 +718,12 @@ module.exports = function(webpackEnv) {
           // The formatter is invoked directly in WebpackDevServerUtils during development
           formatter: isEnvProduction ? typescriptFormatter : undefined,
         }),
+      new (require('monaco-editor-webpack-plugin'))(),
     ].filter(Boolean),
+
+    // support Electron, see https://github.com/facebook/create-react-app/issues/6782 and https://github.com/facebook/create-react-app/pull/5498
+    target: 'electron-renderer',
+    /*
     // Some libraries import Node modules but don't use them in the browser.
     // Tell webpack to provide empty mocks for them so importing them works.
     node: {
@@ -726,6 +736,8 @@ module.exports = function(webpackEnv) {
       tls: 'empty',
       child_process: 'empty',
     },
+    */
+
     // Turn off performance processing because we utilize
     // our own hints via the FileSizeReporter
     performance: false,
